fe/src/test/java/org/apache/doris/analysis/DataDescriptionTest.java

```diff
@@ -54,20 +51,20 @@ public void setUp() {
     @Test
     public void testNormal() throws AnalysisException {
         DataDescription desc = new DataDescription("testTable", null, Lists.newArrayList("abc.txt"),
-                null, null, false, null);
+                null, null, "csv", false, null);
```

Comment:
you don't have to change this file, DataDescription has two constructor to comply with it

Reply:
Ok